commands/javax.py

```diff
@@ -0,0 +1,170 @@
+description:
+    - It is very easy and relatively straight-forward to run a basic Java applications in Ansible using the command or shell modules. Things get complicated and messy pretty quickly when your application needs various options like max heap size or thread stack and has extensive number of class paths and systems properties all being defined on the command line. The javax module helps bring some sanity and readability back to these situations by giving you human-readable argument names and native data structure values to running a Java application from your playbook.
+options:
+    jar:
+        required: false
+        description:
+        - The JAR file that contains the Java class you want to run. Required if the 
+          C(javaclass) option is omitted.
+    javaclass:
+        required: false
+        description:
+        - The main Java class to run. Required if the C(jar) option is omitted.
+    properties:
+        required: false
+        description:
+        - A dict of key value pairs representing system properties to pass the java 
+          application being run. This is the equivalent of the C(-D) commandline 
+          option. 
+    classpath:
+        required: false
+        description:
+        - A list of file system paths the JVM should search to find any 
+          dependencies. The module will take the list and properly concatenate the
+          list of values. The equivalent of the C(-classpath) or C(-cp) commandline 
+          option.
+    init_heap_size:
+        required: false
+        description:
+        - Sets initial Java heap size for the application. Equivalent of the C(-Xms) 
+          'non-stndard' commandline option.  
+    max_heap_size:
+        required: false
+        description:
+        - Sets maximum Java heap size for the application. Equivalent of the C(-Xmx) 
+          'non-stndard' commandline option.  
+    thread_stack:
+        required: false
+        description:
+        - Sets java thread stack size for the application. Equivalent of the C(-Xss) 
+          'non-stndard' commandline option.  
+    executable:
+        required: false
+        default: java
+        description:
+        - Path to java JVM to use. If not supplied,
+          the normal mechanism for resolving binary paths will be used.    
+    java_opts:
+        required: false
+        description:
+        - Specify additional java options not supported by this module by passing 
+          in a list of strings.
+    java_args:
+        required: false
+        description:
+        - Specify a list of strings to be passed as parameters to the java application.
+author: Timothy Appnel
+'''
+
+EXAMPLES = '''
+javax: javaclass='MyMainClass' init_heap_size=1024m max_heap_size=2048m
+args:
+  properties:
+    Name: Foo
+    Hello: World
+  classpath:
+    - /path/to/some.jar
+  java_opts:
+    - '-XX:MaxPermSize=512m'
+    - '-XX:+HeapDumpOnOutOfMemoryError'
+'''
+
```

Comment:
Would be ideal to see a `RETURN` section here. For reference: https://github.com/ansible/ansible-modules-core/blob/devel/packaging/os/apt.py#L141-L161